src/dotnet-install.sh

```diff
@@ -918,15 +918,26 @@ get_http_header_wget() {
     eval $invocation
     local remote_path="$1"
     local disable_feed_credential="$2"
+    local wget_output=''
+    local wget_result=''
 
     remote_path_with_credential="$remote_path"
     if [ "$disable_feed_credential" = false ]; then
         remote_path_with_credential+="$feed_credential"
     fi
 
     wget_options="-q -S --spider --tries 5 --waitretry 2 --connect-timeout 15 "
-    wget $wget_options "$remote_path_with_credential" 2>&1 || return 1
-    return 0
+    wget_output=$(wget $wget_options "$remote_path_with_credential" 2>&1)
+    wget_result=$?
+
+    if [[ $wget_result != 0 ]] && [[ $wget_output == *"unrecognized option:"* ]]; then
+        wget_options="-q -S --spider --tries 5"
```

Comment:
super-nit: Worth keeping the base options in a separate variable to avoid duplicating them on line 929 and here?

Reply:
Should be done with the latest commit now.